packages/cna-template/template/nuxt/pages/index.vue

```diff
@@ -1,6 +1,12 @@
+<%_ if (pug) { _%>
```

Comment:
I think we can put pug templates into a separate dir like `/cna-template/template/frameworks/pug` for avoiding changing existing default template and also having more pug template files.

Reply:
@clarkdo 
Thank you for your review.

The TypeScript branched IF in this file, so I thought this file would be fine.

I fixed to add `/cna-template/template/frameworks/pug` folder!